pkg/actions/commands.go

```diff
@@ -66,8 +66,9 @@ func Commands() {
 					Usage: "Create a project on disk",
 
 					Flags: []cli.Flag{
-						cli.StringFlag{Name: "url, u", Usage: "URL of project to download"},
+						cli.StringFlag{Name: "url, u", Usage: "URL of project to download", Required: false},
```

Comment:
Required: True? Not something you're changing, but I think it needs to have a URL

Reply:
yeah you are right,  thinking about we need to know where to get the template from so it should be true.